lib/elixir/lib/code.ex

```diff
@@ -629,10 +633,8 @@ defmodule Code do
   # unsupported chunk version
   defp lookup_docs(_, _), do: nil
 
-  @doc_sections [:docs, :moduledoc, :callback_docs, :type_docs]
-
   defp do_lookup_docs(docs, :all), do: docs
-  defp do_lookup_docs(docs, kind) when kind in @doc_sections,
+  defp do_lookup_docs(docs, kind) when kind in @doc_kinds,
```

Comment:
ooops.... 
fixed!  thanks for your comments (as usual),
I have created a function head, and removed check in the private functions.
Well, we do need to update it, because this is the full error I was getting: https://gist.github.com/eksperimental/399036a2ccb27fe198e5